### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by tdeekens**
#### Summary

Renovate being on vacation. Greekens steps in.

In honesty: I wanted my weekend work on promster and flopflip in the app.

### CODE REVIEW

File: website-components-playground/package.json [hunk 1]
```diff
@@ -10,23 +10,25 @@
     "serve": "gatsby serve"
   },
   "dependencies": {
+    "@babel/core": "7.7.2",
```
Comment (emmenko) [inline website-components-playground/package.json]: You can remove this

File: website-components-playground/package.json [hunk 2]
```diff
@@ -10,23 +10,25 @@
...
     "@commercetools-frontend/application-components": "15.2.1",
     "@commercetools-frontend/ui-kit": "10.8.0",
     "@emotion/core": "10.0.22",
     "@emotion/styled": "10.0.23",
-    "core-js": "3.4.0",
-    "formik": "2.0.3",
-    "gatsby": "2.17.11",
+    "core-js": "3.4.1",
+    "emotion-theming": "10.0.19",
```
Comment (emmenko) [inline website-components-playground/package.json]: This as well

File: website-components-playground/package.json (hunk 1 above)
Comment (tdeekens) [reply website-components-playground/package.json]: ccfb318

File: website-components-playground/package.json (hunk 2 above)
Comment (tdeekens) [reply website-components-playground/package.json]: ccfb318

**Pull request merged**